src/common/resources/pieceResources.js

```diff
@@ -0,0 +1,35 @@
+import {
+  baseManifest,
+} from '@folio/stripes-acq-components';
+
+import {
+  ORDER_PIECES_API,
+  PIECES_API,
+  CHECKIN_API,
+} from '../constants';
+
+export const pieceResource = {
+  ...baseManifest,
+  accumulate: true,
+  clientGeneratePk: false,
+  fetch: false,
+  path: PIECES_API,
+  records: 'pieces',
+};
+
+export const orderPiecesResource = {
+  ...baseManifest,
+  accumulate: true,
+  clientGeneratePk: false,
+  fetch: false,
```

Comment:
nit: it's not required

Reply:
are you sure?

Reply:
yes, you have accumulate true

Reply:
fetch is default as `true` and it can cause a duplicated request if path is provided.